backend/core/taps/custom.py

```diff
@@ -0,0 +1,54 @@
+from kombu import Connection, Producer, Exchange
```

Comment:
what is the functionality of this tap? Boilerplate code?

Reply:
high-throughput monitor without using real-ones and possible DoS'ing them

Reply:
Great, please add this also in the CHANGELOG and resolve the comment